RackApp/app.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'erb'
+require 'rack'
+require 'active_support/time'
+
+class MyController
+  attr_reader :person, :time
+
+  def initialize(user, time)
+    @person = user
+    @time = time
+  end
+
+  def file(format)
+    File.read(File.join(__dir__, "template.#{format}.erb"))
+  end
+
+  def template(format)
+    ERB.new(file(format))
+  end
```

Comment:
This looks like belonging to another class, name it View or Template

Something like that

```ruby
class MyController
    # This next line could be extracted to a BaseController :)
  class_attribute :template_name, instance_writer: false
  template_name 'template'
  

  def template(format)
    Template.new(template_name, format)
  end

  # rest of the code

 class Template

   def initialize(name, format)
     @erb = file(name, format)
  end
 
  def file(name, format)
     File.read(File.join(__dir__, 'templates', "#{name}.#{format}.erb"))
  end
 end

```